### DISCUSSION TITLE
Anyone missing the PNG info tab?

### DISCUSSION THREAD

**Discussion by slapopotamus** [vladmandic/sdnext#281] [General]
I started using this branch yesterday and I'm hooked. 

The only thing that I'm currently missing is the PNG info tab. Sure I can use the image browser to get the same information but I so used to just dragging rando images from my system to get the prompt information. 

Does anyone have suggestions ? Maybe a local program to pull out prompt information ? 

Are you missing the PNG info tab ?

**Comment by vladmandic** [comment]
this has been mentioned soooo many times - info tab has been merged into process image.
there are still some issues open for that, but that is where it will live for now. if you load image into process image tab, first thing you get is functionality of info tab.

**Reply by slapopotamus** [reply]
Thanks so much for such a quick reply to something that has been asked too many times already.
